### ISSUE TITLE
The weight on regression loss as 5 for these IoU-based losses.

### ISSUE THREAD

**Issue by LIU-JK** [Zzh-tju/DIoU-SSD-pytorch#14]
In this paper,you fix the weight on regression loss as 5 for these IoU-based losses.
Where can I find it？Thank you very much!

**Comment by Zzh-tju** [comment]
tools/train.py

parser.add_argument('--weight', default=5, type=int)

**Comment by LIU-JK** [comment]
1. If the weight value is changed, does ciou have an advantage over other IoU-based losses？

2.If I want to modify the loss function ciou, do I just need to modify the function of / utils / box.utils.py / def bbox _ overlaps_Ciou (bboxes1, bboxes2) ?

**Comment by LIU-JK** [comment]
1If the weight value is changed, does ciou have an advantage over other IoU-based losses？

2.If I want to modify the loss function ciou, do I just need to modify the function of / utils / box.utils.py / def bbox _ overlaps_Ciou (bboxes1, bboxes2) ? @Zzh-tju

**Comment by Zzh-tju** [comment]
1. in high probability
2. yes
